grumphp.yml.dist

```diff
@@ -0,0 +1,21 @@
+# grumphp.yml
+grumphp:
+    tasks:
+        composer:
+            strict: true
+        composer_require_checker: ~
+        phpcsfixer2:
+            allow_risky: true
+            config: .php_cs
+        phpstan:
+            level: 7
```

Comment:
I don't think you need to specify the level here, since it's in the `phpstan.neon` file (generally it's nice to keep that in one place).  I noticed that some of our repos have this here (which is probably where this comes from), but I think that was done from a time when grumphp required that parameter here.